Added a link to resources

joinimran opened the pull request:
Added a link to the resources where the user can get more details related to permissions.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/817

joinimran committed:
Added a link to resources

Added a link to the resources where the user can get more details related to permissions.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/817

joinimran commented:
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

JohanFreelancer9 commented on microsoft-365/compliance/add-or-remove-members-from-a-case-in-advanced-ediscovery.md:
```diff
@@ -18,7 +18,7 @@ description: ""
 # Add or remove members from a case
 
-You can add or remove members to manage who can access the case. Users who are added to Reviewer or eDiscovery Manager role groups can be added as members of the case. The eDiscovery Manager who created the case is automatically added as a member.
+You can add or remove members to manage who can access the case. Users who are added to Reviewer or eDiscovery Manager role groups can be added as members of the case. The eDiscovery Manager who created the case is automatically added as a member. See [assign eDiscovery permissions in the Security & Compliance Center](https://docs.microsoft.com/microsoft-365/compliance/assign-ediscovery-permissio ...
```
```suggestion
You can add or remove members to manage who can access the case. Users who are added to Reviewer or eDiscovery Manager role groups can be added as members of the case. The eDiscovery Manager who created the case is automatically added as a member. See [Assign eDiscovery permissions in the Security & Compliance Center](https://docs.microsoft.com/microsoft-365/compliance/assign-ediscovery-permissions) for more details.
```

JohanFreelancer9 reviewed (changes requested):
Copy Review - minor edit.

joinimran committed:
Update microsoft-365/compliance/add-or-remove-members-from-a-case-in-advanced-ediscovery.md

Co-Authored-By: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

joinimran commented:
@mypil Ready to go live.

mypil commented:
@markjjo - PR has been copy edited and is ready for final review. This is based on issue #817.

Thank you.

cc: @kenwith

markjjo committed:
updated this PR.

Currently, user must be a member of eDiscovery Manager role group to be added as member of a case.
